lng/english.lng.php

```diff
@@ -2127,3 +2126,25 @@
 $lng['admin']['notryfiles']['description'] = 'Say yes here if you want to specify a custom try_files directive in specialsettings (needed for some wordpress plugins for example).';
 $lng['serversettings']['phpfpm_settings']['override_fpmconfig'] = 'Override FPM-daemon settings (pm, max_children, etc.)';
 $lng['serversettings']['phpfpm_settings']['override_fpmconfig_addinfo'] = '<br /><span class="red">Only used if "Override FPM-daemon settings" is set to "Yes"</span>';
+
+
+// FORK
```

Comment:
fork? not if it's part of the froxlor-repo - just follow the naming in the file...or better, just append to the end (no unecessary empty lines please)